pages/_app.tsx

```diff
@@ -1,7 +1,17 @@
-import '../styles/globals.css'
+import '../styles/globals.css';
+import * as Sentry from '@sentry/react';
+import CONFIG from '@config';
+
+if (CONFIG.sentryDSN) {
+  Sentry.init({ dsn: CONFIG.sentryDSN });
+}
 
 function MyApp({ Component, pageProps }) {
```

Comment:
```
App: React.FC = ({children}) => {
  <Sentry.ErrorBoundary>
       <children {...pageProps} />
   </Sentry.ErrorBoundary>
}
```

```
<App>
   <Component />
</App>
```

Reply:
```
import { AppProps } from 'next/app'

function App({ Component, pageProps }: AppProps) {
  return <Component {...pageProps} />
}

export default App
```

This is how it is explained in the next js documentation to convert _app.js to _app.tsx

Reply:
Can you try what I suggested?

Also, please look up React functional components

Reply:
done, if i use <children {...pageProps} /> it shows an error for children component, but i followed React FC way to convert it into tsx.